app/services/recent-events.ts

```diff
@@ -42,6 +43,7 @@ export interface IRecentEvent {
   hash: string;
   isTest?: boolean;
   repeat?: boolean;
+  uuid: string;
```

Comment:
Add a comment that this is a local id and is NOT persistent across app restarts/fetches.